tensorflow_probability/python/bijectors/real_nvp.py

```diff
@@ -144,10 +145,14 @@ def __init__(self,
     """Creates the Real NVP or NICE bijector.
 
     Args:
-      num_masked: Python `int` indicating that the first `d` units of the event
-        should be masked. Must be in the closed interval `[0, D-1]`, where `D`
-        is the event size of the base distribution. If the value is negative,
-        then the last `d` units of the event are masked instead.
+      num_masked: Python `int` or `float` indicating the number of units of the
```

Comment:
I'm not super happy with reusing the argument for this purpose, can we add a second argument (e.g. `fraction_masked` and use that instead?).